collaboration/service/v1alpha/user.proto

```diff
@@ -25,6 +25,7 @@ service UserService {
   // Organization Management
   rpc InviteToOrganization (InviteToOrganizationRequest) returns (InviteToOrganizationResponse);
   rpc RemoveFromOrganization (RemoveFromOrganizationRequest) returns (google.protobuf.Empty);
+  rpc getUserVsGroupDuplicates (GetUserVsGroupDuplicatesRequest) returns (google.protobuf.Empty);
```

Comment:
I think `findDuplicates` will fit better, what do you think?
anyway, should we add a new endpoint or should we enrich the ones that we have, like add a filter/predicate (as we have in other endpoinds) on `ListUsers` in order to find the duplicates?

Reply:
@RuiTeixeira 
Yeah, i am not quite sure where to squeeze that in, as users and user groups are concerned. That's why I wanted to show this somehow in the name. But maybe a new endpoint would be clearer.  

@lzaugg 
Oh thanks, I missed that one! A warm hello back :smile:

Reply:
Decision with Rui: Implement a more general solution with the existing service calls.